odoo/addons/test_new_api/tests/test_one2many.py

```diff
@@ -278,3 +278,22 @@ def test_reward_line_delete(self):
             order.write({
                 'line_ids': [(4, line0.id), (4, line1.id)],
             })
+
+    def test_child_of(self):
+        Team = self.env['test_new_api.team']
+        Member = self.env['test_new_api.team.member']
+
+        team1 = Team.create({'name': 'ORM'})
+        team2 = Team.create({'name': 'Bugfix', 'parent_id': team1.id})
+        team3 = Team.create({'name': 'Support', 'parent_id': team2.id})
+
+        Member.create({'name': 'Raphael', 'team_id': team1.id})
+        member2 = Member.create({'name': 'Noura', 'team_id': team3.id})
+        Member.create({'name': 'Ivan', 'team_id': team2.id})
+
+        # In this specific case...
+        self.assertEqual(member2.id, member2.team_id.parent_id.id)
```

Comment:
A member's id is equal to a team's id?

Reply:
yes

Reply:
You could check STEPS in commit description. To reproduce the problem we need same id values for linked records in different tables

Reply:
The code of the test should be enough.  Add a comment to explain the use-case and the issue that should not occur.

Reply:
I checked description of the commit and comments in the code, but I don't see what could be improved...